### ISSUE TITLE
attention_mask set to None

### ISSUE THREAD

**Issue by longzw1997** [jacklishufan/LaViDa#18]
Thank you for your excellent work! I was wondering if you could kindly explain why attention_mask is set to None at line 296 in llava_dream.py?

**Comment by jacklishufan** [comment]
Hi
we set this because attention mask is never used in training or inference.
At training, we use full attention mask.
At inference, our prefix mask was implemented by the KV Cache, so the variable `attention_mask` will be None
